doc/contribute/external.rst

```diff
@@ -164,3 +170,49 @@ The flowchart below shows an overview of the process:
    :align: center
 
    Submission process
+
+.. _external-tooling:
+
+Contributing External Tooling
+*****************************
+
+This section deals exclusively with the inclusion of external tooling in the
+Zephyr project, where tooling is defined as software that assists the
+compilation, testing or simulation processes but in no case ends up being part
+of the code compiled and linked into the final image.
+
+Tooling components must be released under a license approved by the
+`Open Source Initiative (OSI)`_.
+
+Just like with regular external components, tooling that is imported from
+another project can be integrated either in the main tree or as a :ref:`west
```

Comment:
I'm not sure if this is clear w/regards to tools like a compiler for example, that isn't part of the main tree and isn't reference via west.

Reply:
But then for those tools this page is not applicable at all, because it’s not part of the “default distribution”. The docker image is not considered part of the zephyr distro

Reply:
where do we define default distribution?